CryptoWallet/Extensions/UIButton+.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIButton+.swift
+//  CryptoWallet
+//
+//  Created by Phan Dinh Van on 4/1/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Then
+
+extension UIButton {
+    func setBorder(cornerRadius: CGFloat = 0, borderWidth: CGFloat = 0, borderColor: UIColor) {
+        self.do {
```

Comment:
sorry e chỗ này không cần viết rõ self ra đâu :))
```suggestion
layer.cornerRadius = cornerRadius
layer.borderColor = borderColor.cgColor
layer.borderWidth = borderWidth
clipsToBounds = true
```